src/Preferences.js

```diff
@@ -65,6 +65,15 @@ define(function (require, exports, module) {
                 description: Strings.PREF_INDENT_DESC
             },
             inContextMenu: true
+        }, {
+            id: "autohide",
+            type: "boolean",
+            value: true,
+            options: {
+                name: Strings.PREF_AUTOHIDE_NAME,
+                description: Strings.PREF_AUTOHIDE_DESC
+            },
+            inContextMenu: true
```

Comment:
There is an issue about it, so there seems to be demand, therefore I would absolutely merge it when it is ready.